src/ResourceManager/Common/Commands.Common.Strategies/TargetStateExtensions.cs

```diff
@@ -41,14 +41,14 @@ sealed class Context
 
             public IState Current { get; }
 
-            public string Subscription { get; }
+            public string SubscriptionId { get; }
```

Comment:
I don't think this change is necessary, and it is a breaking change

Reply:
Already updated to 2.0 in @sergey-shandar merged PR https://github.com/Azure/azure-powershell/pull/5129

Reply:
Yes, I'm saying this should remain subscription